### ISSUE TITLE
Adding non seekable streams (url...) throws a warning

### ISSUE THREAD

**Issue by belgattitude** [Ne-Lexa/php-zip#25]
| Q                            | A
| -----------------------------| ---
| Library version(s) affected: | 3.1.8
| PHP version(s):              | 7.1
| OS (with bit depth):         | Ubuntu 18.04 64-bit

**Description**  

In 3.1.8, it's not anymore possible to add remote http streams. There's a warning thrown in ZipNewEntry regarding rewind()...  

```
 public function getEntryContent()
    {
        if (is_resource($this->content)) {
            rewind($this->content);
            return stream_get_contents($this->content);
        }
        return $this->content;
    }
```

http stream is not seekable...

I'll P/R a suggested solution in few minutes to illustrate

**Comment by belgattitude** [comment]
An example would be:

```php
$zipFile = new ZipFile();
$fileUrl = 'https://raw.githubusercontent.com/Ne-Lexa/php-zip/master/README.md';
$fp = @fopen($fileUrl, 'rb', false, stream_context_create([
    'http' => [
        'timeout' => 3,
    ]
]));
if ($fp === false) {
    throw new \Exception(_____);
}

$zipFile->addFromStream($fp, 'remote-file-from-http-stream.md');
$zipFile->saveAsFile($outputZip);
$zipFile->close();
```

A suggested fix is available in #26,

Thanks for having a look and possibily release soon. Great work !!!

**Comment by belgattitude** [comment]
@Ne-Lexa 

you're on full speed ;) Thanks a lot !
